src/main/resources/xlr/CreateAndStartSubRelease.py

```diff
@@ -9,6 +9,11 @@
 
 from xlr.XLReleaseClientUtil import XLReleaseClientUtil
 
+def find_planned_gate_task(tasks):
```

Comment:
[minor] Bit of a confusing name? The method actually just finds planned tasks, it doesn't check that the task is a gate. Also, what if there is no planned gate task?